ng2-components/ng2-activiti-tasklist/src/components/task-header.component.ts

```diff
@@ -207,7 +259,7 @@ export class TaskHeaderComponent implements OnChanges {
     /**
      * Returns true if the task is completed
      */
-    isCompleted() {
+    isCompleted(): boolean {
         return !!this.taskDetails.endDate;
```

Comment:
can you check the end date 
        return this.taskDetails.endDate !== undefined && this.taskDetails.ended !== null;